doc/docs/master/hub/manage-entities/manage-workspaces.md

```diff
@@ -0,0 +1,73 @@
+# Manage Workspaces
+
+In Pachyderm Hub, you run your data science pipelines in workspaces.
+A workspace is a collection of infrastructure resources that run
+Pachyderm components needed to process your data. Workspace resource
+limitations are defined by your account type and provisioned automatically.
+For example, in the free tier, you can only create one workspace, while
+a Pro account allows you to create multiple workspaces.
+
+Each workspace has isolated computing, processing, storage, and other
+resources. Resource autoscaling is managed by Pachyderm behind the
+scenes. You cannot move data or pipelines between workspaces.
+Each workspace in an organization must have a unique name.
+
+## Create a Workspace
+
+You need a workspace to star building your pipelines and working
+with Pachyderm. 
+
+To create a workspace, complete the following steps:
+
+1. Click **Create New Workspace** or **Create a 4-Hour Workspace**.
```

Comment:
This is cleaner and simpler than the "create a workspace" flow you have in the getting started section. I think we should update the other one to be simpler like this